packages/SwingSet/src/liveslots/watchedPromises.js

```diff
@@ -80,7 +80,7 @@ export function makeWatchedPromiseManager(
       promiseRegistrations.delete(vpid);
       for (const watch of watches) {
         const [watcher, ...args] = watch;
-        Promise.resolve().then(() => {
+        void Promise.resolve().then(() => {
```

Comment:
ah, right, if the user-supplied watcher object throws, we want the platform to catch it, and if we added a `.catch()` or something here to satisfy `eslint`, then we wouldn't get that behavior, so `void` is the only appropriate thing to do